spec/classes/document_search_spec.rb

```diff
@@ -743,4 +743,57 @@ def create_documents(document_hashes)
       expect(document_search_results.results.first['content']).to eq 'renew passport'
     end
   end
+
+  describe 'language support' do
+    # Create documents for each supported language
+    languages = [
+      {
+        lang_code: 'en',
+        lang_name: 'English',
+        content: 'Select your state or territory from the dropdown menu to find the rules that apply to you.',
+        query: 'territory'
+      },
+      {
+        lang_code: 'es',
+        lang_name: 'Spanish',
+        content: 'Seleccione su estado o territorio en el menú desplegable y encontrará las normas a seguir.',
+        query: 'territorio'
+      },
+      {
+        lang_code: 'hi',
+        lang_name: 'Hindi',
+        content: 'आप पर लागू होने वाले नियमों को जानने के लिए ड्रॉपडाउन मेनू से अपना राज्य या क्षेत्र चुनें।',
+        query: 'क्षेत्र'
+      },
+      {
+        lang_code: 'bn',
+        lang_name: 'Bengali',
+        content: 'আপনার ক্ষেত্রে প্রযোজ্য নিয়মগুলি খুঁজে পেতে ড্রপডাউন মেনু থেকে আপনার রাজ্য বা অঞ্চল নির্বাচন করুন৷',
+        query: 'অঞ্চল'
+      }
+    ]
+    languages.each do |lang|
+      lang_code, lang_name, content, query = lang.values_at(:lang_code, :lang_name, :content, :query)
+
+      before do
+        create_documents([
+                           {
+                             language: lang[:lang_code],
+                             created: DateTime.now,
+                             path: "https://vote.gov/#{lang_code}",
+                             title: "title #{lang_name}",
+                             description: "description #{lang_name}",
```

Comment:
I would remove `created`, `title` and `description`, as these are not being used by the spec.